db.js

```diff
@@ -4,17 +4,21 @@ const { Sequelize } = require('sequelize');
 
 var getAzureAuthToken = require('./azure-auth')
 
-let ssl;
+let dialectOptions;
```

Comment:
This seems a bit confusing to me. I have the following questions:

1. What is the default case? What happens if MYSQL_CA_CERT and AZURE_CLIENT_ID are not set? Is that what you want?
2. Shouldn't we first create a ssl variable (like in the previous code) and then pass or include it in a dialectOptions afterward? It seems you now have duplicate unnecessary code surrounding the ssl object.
3. Is it ever possible that both MYSQL_CA_CERT and AZURE_CLIENT_ID are true and that they overwrite each other and would that be okay?

Reply:
Yes, changed it!